Mage.Tests/src/test/java/org/mage/test/utils/BoostCountTest.java

```diff
@@ -15,15 +15,15 @@ public class BoostCountTest {
 
     @Test
     public void test_BoostCountSigns() {
-        Assert.assertEquals(CardUtil.getBoostCountAsStr(0, 0), "+0/+0");
-        Assert.assertEquals(CardUtil.getBoostCountAsStr(1, 0), "+1/+0");
-        Assert.assertEquals(CardUtil.getBoostCountAsStr(0, 1), "+0/+1");
-        Assert.assertEquals(CardUtil.getBoostCountAsStr(1, 1), "+1/+1");
+        Assert.assertEquals(CardUtil.getBoostCountAsStrLegacy(0, 0), "+0/+0");
```

Comment:
New method uses same code, so it must be tested too with same use cases for correct +- texts.